oci/oci.go

```diff
@@ -0,0 +1,60 @@
+package oci
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/mrunalp/ocid/utils"
+)
+
+// New creates a new Runtime with options provided
+func New(name string, runtimePath string, sandboxDir string) (*Runtime, error) {
+	r := &Runtime{
+		name:       filepath.Base(runtimePath),
+		path:       runtimePath,
+		sandboxDir: sandboxDir,
```

Comment:
@mrunalp , is there a special reason we want to make `sandboxDir` be a field of `Runtime`, not a field of `Server`?
Will each kubelet node run multiple ocid daemons with different `Runtime` and sandboxDir?